app/assets/stylesheets/responsive.scss

```diff
@@ -1,2 +1 @@
-@import "bootstrap-responsive";
 @import "bootstrap-responsive-overrides";
```

Comment:
Don't need to do it now, but would it make sense to now pull whatever is in the `-overrides` file straight into this one?